### DISCUSSION TITLE
How to Export and Load Fine-Tuned DeepGrow Model in MONAILabel?

### DISCUSSION THREAD

**Discussion by emlased** [Project-MONAI/MONAILabel#1756] [Q&A] [answered]
Hi all,

I have used the Active Learning feature in MONAILabel to fine-tune the DeepGrow model for identifying lung nodules. Now, I would like to export this fine-tuned model and load it onto another MONAILabel server.

I believe the files deepgrow_2d.pt and deepgrow_3d.pt in the radiology/model directory are the most up-to-date versions of my trained model. I tried setting up a new server, creating a radiology/model directory, and copying those files into it, but MONAILabel doesn't seem to point to these files by default.

Could someone guide me on how to correctly load a fine-tuned version of a built-in model (like DeepGrow) into MONAILabel on a new server? Am I missing a step in the configuration?

Thanks in advance for your help!

**Comment by SachidanandAlle** [comment, marked as answer]
Go to models folder in the radiology app and replace the pre trained and currently available deep grow models with your ones..
